packages/client/src/game-core/utils/ResourcesLoader.ts

```diff
@@ -33,17 +33,19 @@ class ResourcesLoader {
                 resolve(this.resourceCache.get(url))
             }
 
-            const img = new Image()
+            if (typeof Image !== 'undefined') {
```

Comment:
Лучше избегать лишних вложенностей, здесь можно сразу выйти из функции при отсутствии Image
```suggestion
            if (typeof Image === 'undefined') {
                return
            }
```